src/libos/src/util/random.rs

```diff
@@ -0,0 +1,15 @@
+use super::*;
+
+use sgx_types::sgx_status_t;
+
+pub fn get_random(rand: &mut [u8]) -> Result<()> {
+    extern "C" {
+        fn sgx_read_rand(rand_buf: *mut u8, buf_size: usize) -> sgx_status_t;
+    }
+
+    let status = unsafe { sgx_read_rand(rand.as_mut_ptr(), rand.len()) };
+    match status {
+        sgx_status_t::SGX_SUCCESS => Ok(()),
+        _ => return_errno!(EAGAIN, "failed to get random number from SGX"),
```

Comment:
If you return EAGAIN, then Occlum would crash.

Reply:
I just move the code from `dev_random.rs` to random.rs, why Occlum will crash?